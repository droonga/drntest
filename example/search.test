{"id":"search:0", "type":"search", "replyTo":"localhost:23003/output", "dataset":"Taiyaki", "body":{"queries":{"result":{"source":"Shops", "condition":{"query":"阿佐ヶ谷", "matchTo":"_key"}, "output":{"elements":["startTime", "elapsedTime", "count", "attributes", "records"], "attributes":["_key"]}}}}}
