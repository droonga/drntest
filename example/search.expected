["droonga.message",0,{"inReplyTo":"search:0","statusCode":200,"type":"proxy.result","body":{"result":{"count":2,"records":[["たいやき工房白家 阿佐ヶ谷店"],["たいやき本舗 藤家 阿佐ヶ谷店"]]}}}]